Campaign/README.md

```diff
@@ -35,7 +35,10 @@ php artisan migrate
 # 5. Generate app key
 php artisan key:generate
 
-# 6. Run seeders (optional)
+# 6. Create elastic structure for campaign stats
```

Comment:
Similarly to the previous comment, this should be part of https://github.com/remp2020/remp/tree/master/Beam#segments-go section.

And as we're trying to add missing indices here, it would be probably OK to create all other indices too. Something like this maybe?

```
curl -XPUT -H "Content-Type: application/json" elastic:9200/events -d '{"mappings": {"_doc": {}}}' 
curl -XPUT -H "Content-Type: application/json" elastic:9200/pageviews -d '{"mappings": {"_doc": {}}}' 
curl -XPUT -H "Content-Type: application/json" elastic:9200/pageviews_time_spent -d '{"mappings": {"_doc": {}}}' 
curl -XPUT -H "Content-Type: application/json" elastic:9200/pageviews_progress -d '{"mappings": {"_doc": {}}}' 
curl -XPUT -H "Content-Type: application/json" elastic:9200/concurrents_by_browser -d '{"mappings": {"_doc": {}}}' 
curl -XPUT -H "Content-Type: application/json" elastic:9200/entities -d '{"mappings": {"_doc": {}}}' 
```

Reply:
i added all of them into Beam readme